app/src/main/java/at/roteskreuz/stopcorona/model/api/ApiInteractor.kt

```diff
@@ -293,4 +308,29 @@ sealed class SicknessCertificateUploadException : Exception() {
      * Triggered in case of an SMS Gateway error.
      */
     object SMSGatewayException : SicknessCertificateUploadException()
-}
\ No newline at end of file
+}
+
+/**
+ * Collection of downloaded diagnosis archives
+ */
+data class ListOfDailyBatches(
+    val diagnosisArchiveFilesOfTheDay: MutableList<ArchivesOfOneDay> = ArrayList()
+)
+
+/**
+ * One day worth of diagnosis key files
+ */
+data class ArchivesOfOneDay(
+    /**
+     * path to the diagnosis key files downloaded from the backend as local temp files
+     */
+    val archiveFilePaths: List<File>,
+    /**
+     * unix timestamp of the day
+     */
+    val dayTimestampOfDay: Long,
```

Comment:
Convert it to the correct timezone. @dusanjencik

Reply:
Unix time stamp is always UTC